### PR TITLE
docs: ✏️ Adding roadmap section

### PR DESCRIPTION
**Pull request by nicolaspoggi**
Added a new page with the high-level roadmap information, that we will post soon on the blog.

This is not to be merged right away, just reviewed and approved (estimated merge is end of day today).

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+date: 2023-01-10T09:00:00.000+00:00
+description: Fleek Network's high level roadmap per stage. Devnet, Testnet, and Mainnet.
+category: Documentation
+keywords: [roadmap]
+tags:
+- Roadmap
+- Fleek Network
+---
+
+![Fleek Network: Roadmap](./images/fra.png?202301101924)
```
Comment (heldrida) [inline docs/roadmap.md:13]: Mind rename the file and provide something a bit more sound, please?

File: docs/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Introduction
+
+This is a high-level roadmap overview of Fleek Network's development updated on `February 2023`. The roadmap is divided in three phases:
```
Comment (heldrida) [inline docs/roadmap.md:17]:
```suggestion
This is a high-level roadmap overview of Fleek Network's development, updated on `February 2023`. The roadmap is divided into three phases:
```

File: docs/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Devnet**: Focused on the building of the Network's core features and architecture.
+- **Testnet**: Focused on the testing of the Network's individual components.
+- **Mainnet**: Stable and production release of the protocol, fully featured and tested.
+
+## Devnet
+During Devnet, the current stage, the team will set the foundational components needed to build Fleek Network's decentralize edge, including the following:
```
Comment (heldrida) [inline docs/roadmap.md:24]:
```suggestion
During Devnet, the current stage, the team will set the foundational components needed to build Fleek Network's decentralized edge, including the following:
```

File: docs/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+- The network actors
+    - Cache and Gateway Nodes
+        - Full chain nodes, validators, and light nodes
+            - Full Chain nodes: maintain history but are not necceseraily validators in the network.
```
Comment (heldrida) [inline docs/roadmap.md:29]:
```suggestion
            - Full Chain nodes: maintain history but are not necessarily validators in the network.
```

File: docs/roadmap.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            - Validators: don’t have to maintain the full network history.
+            - Light nodes: act as light verification nodes. 
+- Content Routing and Replication
+- Reputation System
+- Modular Consensus Algorithm
+    - Narwhal and Bullshark
+- Network Governance and Economics
+- EVM Compatiable application layer
```
Comment (heldrida) [inline docs/roadmap.md:37]:
```suggestion
- EVM Compatible application layer
```

File: docs/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+- L1 <> L2 <> Fleek Network Bridge
+- SDK Libraries (Rust / Typescript)
+
+
+## Testnet
+During Testnet, the focus is twofold:
+
+- Stress-test and iterate Fleek Network
+- Maximize number of operators, nodes, and geographical distribution.
```
Comment (heldrida) [inline docs/roadmap.md:46]:
```suggestion
- Maximize the number of operators, nodes, and geographical distribution.
```

File: docs/roadmap.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+The team will develop different testnet versions, **test-specific**, to individually test, stress, and validate the network's performance and design. These are some of the items we intend to test:
+
+- Consensus
+- Performance
+- Incentives
+
+We also intend to **integrate IPFS/Filecoin** to allow direct file-fetching from IPFS an Filecoin nodes through Fleek Network (aside from the network's direct HTTPs capabilities).
```
Comment (heldrida) [inline docs/roadmap.md:54]:
```suggestion
We also intend to **integrate IPFS/Filecoin** to allow direct file-fetching from IPFS and Filecoin nodes through Fleek Network (aside from the network's direct HTTPs capabilities).
```

File: docs/roadmap.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+## Mainnet
+
+The mainnet release of Fleek Network will come with the arrival to a stable and end-to-end tested version of the protocol. The transition to mainnet depends on u
```
Comment (heldrida) [inline docs/roadmap.md:58]:
```suggestion
The mainnet release of Fleek Network will come with the arrival of a stable and end-to-end tested version of the protocol. The transition to mainnet depends on you!
```

**Review by heldrida — CHANGES_REQUESTED** [review]

**Comment by heldrida** [comment]
@nicolaspoggi PR should be set against "develop" and not "main".

File: docs/roadmap.md (hunk 1 above)
Comment (nicolaspoggi) [reply docs/roadmap.md]: Of course! You're right :D

**Comment by nicolaspoggi** [comment]
Thanks for the reviews @heldrida, updated image and comments :)

**Review by heldrida — APPROVED** [review]

**Pull request merged**